base/client.jl

```diff
@@ -66,15 +73,28 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
-        @static if !Sys.iswindows()
+        local command::Cmd
+        if Sys.iswindows()
+            if shell_name == ""
+                command = cmd
+            elseif shell_name == "cmd"
+                command = Cmd(`$shell /c $(shell_escape_CMDly(shell_escape_winsomely(cmd)))`, windows_verbatim=true)
+            elseif shell_name in ("powershell", "pwsh")
+                command = Cmd(`$shell -Command $(shell_escape_PWSHly(shell_escape_winsomely(cmd)))`, windows_verbatim=true)
+            elseif shell_name == "busybox"
```

Comment:
Is this really necessary ? We don't even ship with busy box anymore...

Reply:
arg .. looks like it is needed for testing on windows...  maybe we should look into WSL  for testing on windows instead of downloading busybox?

Reply:
Still something about this rubs me the wrong way. We should still try to remove this and try to work around them in the tests. Thoughts welcome

Reply:
I think we should make this the `else` block

Reply:
What exactly do you mean by "making this the else block"? 
Do you mean :

```julia
           if shell_name == "cmd"
                command = _CMD_execute(shell, cmd)
            elseif shell_name in ("powershell", "pwsh")
                command = _powershell_execute(shell, cmd)
            else
                  command  =  `$shell sh -c $(shell_escape_posixly(cmd))`  
```
Is the assumption here that if you don't want `cmd` or `powershell`  then you probably want it to default to use a  linux-based shell running in a windows environment?